snowpack/src/build/process.ts

```diff
@@ -234,6 +231,11 @@ async function flushFileQueue(
       continue;
     }
     const result = await devServer.loadUrl(fileUrl, loadOptions);
+    if (!result) {
```

Comment:
… but when building, try and build all possible URLs, and fail silently if output wasn’t generated (in other words, try and build a `.css` for every `.svelte` file, but don’t throw an error for the Svelte files that don’t return this.).